.github/workflows/main.yaml

```diff
@@ -24,6 +24,11 @@ on:
         type: string
         description: Name of the directory to use at https://javadocs.sonarsource.org/ when publishJavadoc is set to true
         required: false
+      publicRelease:
+        type: boolean
+        description: Indicate if the project is generating a public release or not
```

Comment:
If this is false by default, wouldn't it affect the existing repos that use publish Javadoc publicly?

Reply:
Hi @jayadeep-km-sonarsource yes however I checked and none of our organization repositories are using it at the moment (Also the majority of our repositories are private so I believe false is good as default)